dvc/command/git_hook.py

```diff
@@ -1,36 +1,37 @@
-import logging
-import os
-
 from dvc.command.base import CmdBaseNoRepo, fix_subparsers
-from dvc.exceptions import NotDvcRepoError
-
-logger = logging.getLogger(__name__)
 
 
 class CmdHookBase(CmdBaseNoRepo):
+    cmd = None
+    collect_analytics = False
```

Comment:
Arguably git-hook commands should also trigger the analytics. It is a piece of pretty useful knowledge for us.

Reply:
Personally, I think it's too much to run anything else on git hooks. Sure, we can collect details if user are using pre-commit or not and how often they do, etc. But, collecting and sharing them at that instant feels wrong.

Reply:
I think it's acceptable for us to run the analytics on git hooks given that users can opt out and disable them. Maybe we could just add a note in the docs that analytics will also be collected when dvc is run via the pre-commit hooks?

Reply:
@pmrowla Not sure it is worth documenting explicitly, because we already say that we just collect the  analytics when you run our commands, and git-hook is a command too, so it seems logical that it wouldn't be an exception.